documentation/blog/2025-05-22-llm-agent-readiness/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: 3 Prompts to Test for Agent Readiness
+description: A series of prompts to test an LLM's capabilities to be used with AI agents
+authors: 
+    - angie
+---
+
+![blog cover](llm-agent-test.png)
+
+[Goose](/) is LLM-agnostic, meaning you can plug in the model of your choice. However, not every LLM is suitable to work with agents. Some may be great at *answering* things, but not actually *doing* things. If you're considering which model to use with an agent, these 3 prompts can quickly give you a sense of the model's capabilities.
+
+<!-- truncate -->
+
+## Tool Calling
+
+This initial prompt tests for tool calling capabilities. Its ask is forceful to reduce a shy model's hesitation to make function calls.
+
+```bash
+Create a file at ~/workspace/loose-goose/tool-test.txt with the contents "Hello World".
+
+Use the write tool. Do not ask for confirmation. Just do it.
+```
+
+✅ tool-test.txt was created
+
+❌ the agent responds by telling you the code to write yourself
+
+**Example of successful response**
+
+```bash
+─── text_editor | developer ──────────────────────────
+path: ~/workspace/loose-goose/tool-test.txt
+command: write
+file_text: Hello World
+
+The file has been created successfully with the following content:
+
+"Hello World"
+```
+
+The model emits a structured tool call in JSON.
+
+## Memory Awareness
+
+Next, I test to see if the agent can recall what it's doing. It's critical that the model can remember previous actions and continues logically.
```

Comment:
For most of the blog post, you don't use first person, but then you use it here. Suggestion for consistency:
```suggestion
Next, test whether the agent can recall what it’s doing.It's critical that the model can remember previous actions and continues logically.
```